src/CreateBoardForm.tsx

```diff
@@ -2,51 +2,30 @@ import React, { Component, FormEvent } from 'react'
 import { remove, shuffle } from 'lodash'
 import { Board } from './Board'
 import { Checkbox } from './Checkbox'
+import { database } from './firebaseConfig'
 
-export class CreateBoardForm extends Component<{ togglePage: any }, { chosenCharacters: string[], showBoard: boolean, board: string[], boardName: string }> {
-  baseCharacters: string[] = [
-    "Arya Stark",
-    "Sansa Stark",
-    "Bran Stark",
-    "Jon Snow",
-    "Theon Greyjoy",
-    "Yara Greyjoy",
-    "Euron Greyjoy",
-    "Cersei Lannister",
-    "Jaime Lannister",
-    "Tyrion Lannister",
-    "Sam Tarly",
-    "Tormund Giantsbane",
-    "Brienne of Tarth",
-    "Gendry",
-    "Podrick",
-    "Danaerys Targaryen",
-    "Jorah Mormont",
-    "Davos Seaworth",
-    "Lyanna Mormont",
-    "The Night King"
-  ]
+export class CreateBoardForm extends Component<{ togglePage: any }, { chosenCharacters: string[], showBoard: boolean, board: string[], boardName: string, baseCharacters: string[], optionalCharacters: string[] }> {
```

Comment:
it'd be more readable if you moved this type definition above the class so you could break it into multiple lines.

Reply:
yeah that's a good point. I'll do that as a next thing I do